tests/common/devices.py

```diff
@@ -647,6 +649,79 @@ def check_bgp_session_state(self, neigh_ips, neigh_desc, state="established"):
 
         return False
 
+    def exec_template(self, ansible_root, ansible_playbook, **kwargs):
+        playbook_template = 'cd {ansible_path}; ansible-playbook {playbook} -i lab -l {fanout_host} --extra-vars \'{extra_vars}\' -vvvvv'
```

Comment:
Do not hardcode inventory file name here

Reply:
Fixed.